<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Nita-pc" name="Default test" tests="1" failures="0" timestamp="2 May 2018 14:47:35 GMT" time="20.97" errors="0">
  <testcase name="@BeforeClass configBC" time="0.0" classname="com.actitime.testscript.CreateCustomer">
    <failure type="java.io.FileNotFoundException" message=".\dataData\TestData.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: .\dataData\TestData.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at com.actitime.genericLib.CommonUtility.getpropertiesData(CommonUtility.java:23)
	at com.actitime.genericLib.BaseClass.configBC(BaseClass.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass configBC -->
  <testcase name="@BeforeMethod configBM" time="0.0" classname="com.actitime.testscript.CreateCustomer">
    <skipped/>
  </testcase> <!-- @BeforeMethod configBM -->
  <testcase name="@AfterMethod configAM" time="0.0" classname="com.actitime.testscript.CreateCustomer">
    <skipped/>
  </testcase> <!-- @AfterMethod configAM -->
  <testcase name="@AfterClass configAC" time="0.0" classname="com.actitime.testscript.CreateCustomer">
    <skipped/>
  </testcase> <!-- @AfterClass configAC -->
  <testcase name="CreatTest" time="0.029" classname="com.actitime.testscript.CreateCustomer">
    <skipped/>
  </testcase> <!-- CreatTest -->
</testsuite> <!-- Default test -->
